Assets/Mirror/Components/InterestManagement/Match/MatchInterestManagement.cs

```diff
@@ -7,129 +7,123 @@ namespace Mirror
     [AddComponentMenu("Network/ Interest Management/ Match/Match Interest Management")]
     public class MatchInterestManagement : InterestManagement
     {
-        readonly Dictionary<Guid, HashSet<NetworkIdentity>> matchObjects =
-            new Dictionary<Guid, HashSet<NetworkIdentity>>();
+        readonly Dictionary<Guid, HashSet<NetworkMatch>> matchObjects =
+            new Dictionary<Guid, HashSet<NetworkMatch>>();
 
-        readonly Dictionary<NetworkIdentity, Guid> lastObjectMatch =
-            new Dictionary<NetworkIdentity, Guid>();
+        readonly Dictionary<NetworkIdentity, NetworkMatch> lastObjectMatch =
```

Comment:
Could be removed entirely I believe: https://github.com/MirrorNetworking/Mirror/commit/b900b2e00a28cacc0ee1ad29fedfe6ee46d78cb6

Reply:
Merged your changes